netcontrold/lib/dataif.py

```diff
@@ -570,10 +611,8 @@ def pmd_load(pmd):
     # cycles of a pmd, calculate load on this pmd.
     # sort counters so that, incremental differences calculated.
     sort_rx_cyc = sorted(pmd.rx_cyc[:])
-    sort_idle_cyc = pmd.idle_cpu_cyc[:]
-    sort_idle_cyc.sort()
-    sort_proc_cyc = pmd.proc_cpu_cyc[:]
-    sort_proc_cyc.sort()
+    sort_idle_cyc = sorted(pmd.idle_cpu_cyc[:])
```

Comment:
you added sort_idle_cyc but did not use it. You missed some calculation or  it is type ?
Similarly sort_proc_cyc as well.

Reply:
I said about additional changes made by autopep8 right..! these are them.... instead of assigning and sorting in 2 different lines.. it is making them to 1 single line

Reply:
ack